WordPress/Classes/ViewRelated/Post/PostCardStatusViewModel.swift

```diff
@@ -65,6 +65,8 @@ class PostCardStatusViewModel: NSObject {
             return NSLocalizedString("Uploading post...", comment: "Message displayed on a post's card when the post has failed to upload")
         } else if !post.hasLocalChanges() && post.hasAutosaveRevision {
             return StatusMessages.hasUnsavedChanges
+        } else if post.hasVersionConflict() {
+            return StatusMessages.hasVersionConflict
```

Comment:
We have some tests for the messages, you can see them here: https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/WordPressTest/ViewRelated/Post/Views/PostCardStatusViewModelTests.swift#L106

We just need to manipulate the post so it has an `.original` with a different date.